src/schemas/xspec.rnc

```diff
@@ -37,7 +37,34 @@ description =
 		[ a:defaultValue = "import" ]
 		attribute run-as { "import" | "external" }?,
 		
-		(global-param | import | scenario | pending | variable)+ }
+		(helper | global-param | import | scenario | pending | variable)+ }
+
+helper =
+	## The test runner will import this stylesheet (XSLT) or query (XQuery) module after importing
+	## the SUT module.
```

Comment:
It seems that this feature works with `run-as="import"` (the default) and not `run-as="external"`. Should this description say anything about that? In any case, whenever the wiki documents this feature, I think it's worth mentioning that to set the scope.

Reply:
`run-as=external` allows helper components to be used in some places. I added a test in 706f61e43f2fdb6a2dbca3f080f273b7e41f9ba1.

To summarize,

- `run-as=import`:
  - Helpers can be used in most places.
  - Helper templates are applied to the context set up by `x:context`.
  - Helper stylesheets can override SUT. You would need to be careful not to interfere with SUT too much.
- `run-as=external`:
   - Helpers do not work in `x:call/(@function|@template)`.
   - Helper templates are not applied to the context.
   - Helpers do not affect SUT.

Reply:
Glad to hear the feature is more applicable to the external situation than I had thought at first.

When you adapt your comment above for use in the wiki, can you clarify the point about being careful not to interfere with the SUT "too much"? Would it be clearer for the wiki to say something about ensuring that overrides are suitable and do not have adverse side effects in your test? Maybe that is still vague.

Reply:
Good point. How about this?

> * Helper stylesheets can override the test target stylesheet.
>   For example, if you want to change the behavior of the test target for the purposes of unit testing, you can write some templates in a helper stylesheet and override the test target stylesheet. That's exactly the same as what you usually do using `xsl:import`.
>   Make sure your helper module fulfills only a specific and limited purpose, otherwise your helper may produce adverse side effects on the behavior of the test target and turn your tests meaningless. Likewise the test target may also have side effects on your helper and you may need to protect the helper against it ([example](https://github.com/xspec/xspec/blob/5b01d612b142afead4e438812d56c46f0c4356f8/tutorial/helper/ws-only-text/test-helper.xsl#L21-L24)).

Reply:
I like it. Just a few editorial-type suggestions in the last paragraph:
Make sure your helper module fulfills only a specific and limited purpose. Otherwise, your helper may produce adverse side effects on the behavior of the test target and make your tests misleading. Likewise, the test target...

Reply:
Thanks for the edit. I took it and pushed https://github.com/xspec/xspec/wiki/Integrating-Your-Own-Test-Helpers